### ISSUE TITLE
cross-unzip doesn't fail on Windows if the PowerShell command fails

### ISSUE THREAD

**Issue by GMZwinge** [feross/cross-zip#19]
Found the issue when one of the extracted file used a path longer than the Windows limit (around 260 characters).

**Comment by GMZwinge** [comment]
It seems that changing
```
      '-command', '& { param([String]$myInPath, [String]$myOutPath); Add-Type -A "System.IO.Compression.FileSystem"; [IO.Compression.ZipFile]::CreateFromDirectory($myInPath, $myOutPath); }',
```
into
```
      '-command', '& { param([String]$myInPath, [String]$myOutPath); Add-Type -A "System.IO.Compression.FileSystem"; [IO.Compression.ZipFile]::CreateFromDirectory($myInPath, $myOutPath); exit !$? }',
```
may be enough.

**Comment by GMZwinge** [comment]
There may be a similar issue with the cross-zip.

**Comment by GMZwinge** [comment]
To repro with path longer than 260 characters, make sure the registry entry LongPathsEnabled from https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file is not set.

**Comment by GMZwinge** [comment]
Just in case, it's on Windows 10 version 1903 and PowerShell version 5.1.18362.145.

**Comment by malept** [comment]
I found an easier way to trigger this: try to unzip a file where the path contains a space.

**Comment by malept** [comment]
I've created a pull request with the exit changes in #22.

**Comment by feross** [comment]
Released as 3.1.0
